ci-operator/config/medik8s/node-maintenance-operator/medik8s-node-maintenance-operator-main.yaml

```diff
@@ -24,7 +24,7 @@ releases:
     candidate:
       product: ocp
       stream: ci
```

Comment:
In NHC and SNR we use `nightly` stream. Looking [here](https://stackoverflow.com/questions/417134/continuous-integration-vs-nightly-builds#:~:text=The%20CI%20build%20is%20a,is%20deemed%20ready%20for%20testing.) it seems like it would be better to use `nightly` stream over here as well. 
WDYT @slintes @mshitrit ?